rpc/http.go

```diff
@@ -65,21 +65,14 @@ func (hc *httpConn) Close() error {
 
 // DialHTTP creates a new RPC clients that connection to an RPC server over HTTP.
 func DialHTTP(endpoint string) (*Client, error) {
```

Comment:
is verifying the endpoint is a legal string necessary? just a thought

Reply:
maybe it's already done in the caller, have you checked?